lib/Models/Cesium.ts

```diff
@@ -239,27 +239,17 @@ export default class Cesium extends GlobeOrMap {
         );
       }
       if (expandLink) {
-        let attributionToAboutPage = document.createElement("div");
-        attributionToAboutPage.innerHTML = `<a href="about.html#data-attribution" target="_blank" rel="noopener noreferrer">Data attribution</a>`;
-        let disclaimerToAboutPage = document.createElement("div");
-        disclaimerToAboutPage.innerHTML = `<a href="about.html#disclaimer" target="_blank" rel="noopener noreferrer">Disclaimer</a>`;
-
-        if (logoContainer && logoContainer.parentNode) {
-          if (disclaimerToAboutPage && disclaimerToAboutPage.firstChild) {
-            logoContainer.parentNode.insertBefore(
-              disclaimerToAboutPage.firstChild,
-              logoContainer.nextSibling
-            );
-          }
-
-          if (attributionToAboutPage && attributionToAboutPage.firstChild) {
-            logoContainer.parentNode.insertBefore(
-              attributionToAboutPage.firstChild,
-              logoContainer.nextSibling
-            );
-          }
-        }
-
+        this.terria.configParameters.extraCreditLinks
+          ?.reverse()
+          .forEach(({ url, text }) => {
+            // Create a link and insert it after the logo node
+            const el = document.createElement("div");
+            el.innerHTML = `<a href="${url}" target="_blank" rel="noopener noreferrer">${text}</a>`;
+            const linkNode = el.firstChild as Element | null;
```

Comment:
instead of doing this, it might be easier to create an 'a' element and appendChild to your el element, that way you'll know for sure it exists and can get rid of the null checks

Reply:
> Just for my understanding, how come we are reaching into the dom and adding elements outside of React?

You are right. I think the reason why it was done that way originally is that we are dealing with DOM elements created by Cesium. We could still separate it out as a renderless component. I have created an issue for it here - https://github.com/TerriaJS/terriajs/issues/5584

Reply:
Makes sense. I think we don't need the `el` if we create `a` directly (but you loose the convenience of innerHTML). I have updated the code to do so.